thrust/thrust/system/cuda/detail/core/triple_chevron_launch.h

```diff
@@ -29,6 +29,9 @@
 #include <thrust/detail/config.h>
 #include <thrust/system/cuda/detail/core/alignment.h>
 #include <thrust/system/cuda/detail/guarded_cuda_runtime_api.h>
+
+#include <cuda/std/detail/__config>
+
 #include <cassert>
```

Comment:
Maybe this would be better?

```suggestion
#include <cuda/std/cassert>
```